graphql_compiler/schema_generation/sqlalchemy/scalar_type_mapper.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2019-present Kensho Technologies, LLC.
```

Comment:
I've seen and reviewed this file in `add-sqlalchemy-schema-generation` feature branch. Assuming no changes.

Reply:
Correct. No changes.